acceptance/tests/facts/fedora.rb

```diff
@@ -1,80 +1,90 @@
-                  'os.release.major'        => os_version,
-                }
+    step "Ensure the OS fact resolves as expected" do
+      expected_os = {
+          'os.architecture'  => os_arch,
+          'os.family'        => 'RedHat',
+          'os.hardware'      => os_hardware,
+          'os.name'          => /Fedora/,
+          'os.release.full'  => os_version,
+          'os.release.major' => os_version,
+      }
 
-  expected_os.each do |fact, value|
-    assert_match(value, fact_on(agent, fact))
-  end
+      expected_os.each do |fact, value|
+        assert_match(value, fact_on(agent, fact))
+      end
+    end
 
-  step "Ensure the Processors fact resolves with reasonable values"
-  expected_processors = {
-                          'processors.count'         => /[1-9]/,
-                          'processors.physicalcount' => /[1-9]/,
-                          'processors.isa'           => os_hardware,
-                          'processors.models'        => /"Intel\(R\).*"/
-                        }
+    step "Ensure the Processors fact resolves with reasonable values" do
+      expected_processors = {
+          'processors.count'         => /[1-9]/,
+          'processors.physicalcount' => /[1-9]/,
+          'processors.isa'           => os_hardware,
+          'processors.models'        => /"Intel\(R\).*"/
+      }
 
-  expected_processors.each do |fact, value|
-    assert_match(value, fact_on(agent, fact))
-  end
+      expected_processors.each do |fact, value|
+        assert_match(value, fact_on(agent, fact))
+      end
+    end
 
-  step "Ensure the identity fact resolves as expected"
-  expected_identity = {
-                        'identity.gid'        => '0',
-                        'identity.group'      => 'root',
-                        'identity.uid'        => '0',
-                        'identity.user'       => 'root',
-                        'identity.privileged' => 'true'
-                      }
+    step "Ensure the identity fact resolves as expected" do
+      expected_identity = {
+          'identity.gid'        => '0',
+          'identity.group'      => 'root',
+          'identity.uid'        => '0',
+          'identity.user'       => 'root',
+          'identity.privileged' => 'true'
+      }
 
-  expected_identity.each do |fact, value|
-    assert_equal(value, fact_on(agent, fact))
-  end
+      expected_identity.each do |fact, value|
+        assert_equal(value, fact_on(agent, fact))
+      end
+    end
 
-  step "Ensure the kernel fact resolves as expected"
-  kernel_version = case os_version
-    when '20' then '3.11'
-    when '21' then '3.17'
-    when '22' then '4.0'
-    else 'undefined'
-  end
+    step "Ensure the kernel fact resolves as expected" do
+      kernel_version = case os_version
+                         when '20' then
+                           '3.11'
+                         when '21' then
+                           '3.17'
+                         when '22' then
```

Comment:
`then` is only needed in single-line `when` statements, so it's not needed when you break it into multiple lines. It doesn't hurt anything to have them, but for future reference.